samples/snippets/pom.xml

```diff
@@ -63,6 +63,19 @@
       <artifactId>google-cloud-storage</artifactId>
     </dependency>
 
+    <!-- AWS SDK for Java V2 -->
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>auth</artifactId>
+      <version>2.20.27</version>
+    </dependency>
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>regions</artifactId>
+      <version>2.20.27</version>
+    </dependency>
```

Comment:
nit: Can we import their BOM? and have the bom manage these versions together?

I know above we seem to have a bom but manage some deps separately (I'm not too sure why that's the case).